libtracepoint-decode-cpp/src/PerfDataFile.cpp

```diff
@@ -621,14 +623,16 @@ PerfDataFile::ReadEvent(_Outptr_result_maybenull_ perf_event_header const** ppEv
                     case PERF_HEADER_CLOCK_DATA:
                         ParseHeaderClockData();
                         break;
-                    case PERF_HEADER_LAST_FEATURE:
-                        ParseHeaderEventDesc();
-                        break;
```

Comment:
Note: Some perf.data files do not have a LAST_FEATURE event, so the EventDesc header was never getting parsed. Move parsing to the FINISHED_INIT event, which is more consistently present.